tools/bridge/bridge/constants.py

```diff
@@ -3,3 +3,5 @@
 TRANSFER_EVENT_NAME = "Transfer"
 CONFIRMATION_EVENT_NAME = "Confirmation"
 COMPLETION_EVENT_NAME = "TransferCompleted"
+
+CONFIRMATION_TRANSACTION_GAS_LIMIT = 300_000
```

Comment:
What is this value? Please add a short comment describing this value!

The last confirmation that completes a transaction currently uses 320821 gas with 123 validators, so if this value is related to that you also have to increase it a bit.

You can run `test_measure_gas_home_bridge.py` to show you the actual values.